projects/plugins/jetpack/_inc/lib/class-jetpack-ai-helper.php

```diff
@@ -55,6 +55,22 @@ public static function get_status_permission_check( $request ) { // phpcs:ignore
 		return true;
 	}
 
+	/**
+	 * Return true if these features should be active on the current site.
+	 * Currently, it's limited to WPCOM Simple and Atomic.
+	 */
+	public static function is_enabled() {
+		$default = false;
+
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+			$default = true;
+		} elseif ( ( new Automattic\Jetpack\Status\Host() )->is_woa_site() ) {
+			$default = true;
+		}
+
+		return apply_filters( 'jetpack_ai_enabled', $default );
```

Comment:
```suggestion
		/**
		 * Filter whether the AI features are enabled in the Jetpack plugin.
		 *
		 * @since $$next-version$$
		 *
		 * @param bool $default Are AI features enabled? Defaults to false.
		 */
		return apply_filters( 'jetpack_ai_enabled', $default );
```